### DISCUSSION TITLE
es/tasks/detect/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#14642] [Docs]
# es/tasks/detect/

Aprende sobre la detección de objetos con YOLOv8. Explora los modelos preentrenados, el entrenamiento, la validación, la predicción y los detalles de exportación para un reconocimiento eficaz de objetos.

https://docs.ultralytics.com/es/tasks/detect/

**Comment by wisrovi** [comment]
he intentado realizar una inferencia de un video, pero este va muy lento, porque esta procesando frame a frame, tengo 24GB de GPU y solo usa 2.5GB, por lo que tengo mucha GPU disponible, pero no consigo usarla toda (o casi toda) para acelerar mi inferencia

me podrian indicar si hay alguna manera de que pueda usar toda la GPU para inferencia?

gracias

**Reply by pderrenger** [reply]
@wisrovi para mejorar la velocidad de inferencia en video y aprovechar al máximo tu GPU, te recomiendo verificar si estás utilizando la última versión del paquete Ultralytics. Además, puedes intentar ajustar el tamaño del lote (batch size) durante la inferencia para procesar múltiples frames simultáneamente. Esto puede ayudar a utilizar más recursos de la GPU y acelerar el proceso. Si el problema persiste, por favor proporciona más detalles sobre tu configuración actual para poder ofrecerte una mejor asistencia. Gracias por tu consulta.
